protos/core/Btfs.proto

```diff
@@ -0,0 +1,256 @@
+syntax = "proto3";
+
+option java_multiple_files = true;
+option java_package = "org.tron.protos"; //Specify the name of the package that generated the Java file
+option java_outer_classname = "Protocol"; //Specify the class name of the generated Java file
+
+
+package btfs;
+
+
+service guard {
+
+  	rpc startContract (Contract) returns (Contract) {}
+  	rpc checkContract(CheckContractRequest) returns (ContractStatus){}
+  	rpc reportUploadStatus(UploadStatusReport) returns(Result){}
+  	rpc updatePayment(UpdatePaymentRequest) returns (Result){}  //reserved, notification while payment has been changed
+  	rpc health() returns(HealthStatus){}
+}
+
+//service provided by escrow need to be request by guard
+service escrow{
+	rpc reserveFund(ProofOfFund) returns (ProofOfFund){}
+	rpc startEscrow(StartEscrowRequest) returns (PaymentStatus){}
+	rpc updatePayment(UpdatePaymentRequest) returns (PaymentStatus){}  //upload payment status and change host were combined in this rpc
+
+}
+
+//host provided service
+service host{
+	rpc challenge(ProofOfReplicateChallenge) returns (ProofOfReplicateChallenge){}
+}
+
+
+
+
+message ProofOfFund{
+	bytes id = 1;// the proof id provide by escrow
+	bytes reference_id = 2; //the reference id to present purpose of such proof, in the BTFS, it is file_hash or chunk_hash
+	bytes escrow_address = 3; //the escrow service’s public key 
+	bytes owner_address = 4; //owner’s public key
+	bytes target_address = 5; //payment target’s pub key, in BTFS, it is host pub key for renter prepare such POF, and is renter pub key for host to prepare such POF
```

Comment:
This will not filled by escrow

Reply:
This field will be filled by the renter, and from my understanding, only id, signed_time, escrow_signature were filled by escrow service, not sure we can meet understanding.